packages/edit-post/src/components/options-modal/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { PostTaxonomies } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import { EnablePublishSidebarOption, EnableTipsOption, EnablePanelOption } from './options';
```

Comment:
I think it aids with comprehension to have these closely related components declared in the same source file. _One component per file_ is a nice guideline but in this case I think it would just annoy the reader to make them have to click around a lot. 

With regards to the names, I like that they currently describe the UI that's shown. For example, the `EnableTipsOption` is literally the _Enable Tips_ option:

<img width="440" alt="screen shot 2018-10-16 at 13 33 31" src="https://user-images.githubusercontent.com/612155/46989241-1978d580-d148-11e8-8471-9e58528911a7.png">

I'm happy to concede on any of this if you feel strongly about it! 🙂

Reply:
Well, it depends on the reader. We aren't consistent at all in the codebase. Let's keep it as is in one file :)

We don't have strict rules for component names. I personally try to avoid using verbs at the beginning of the component as it implies action happening. I don't feel strongly about it, too. Your reasoning is also correct.

Reply:
Naming things, hey! 😅